### PR TITLE
Implement About us sections

### PR DESCRIPTION
**Pull request by srgzlb**
This PR's purpose is to implement the About us section and since the Preview app is available on prod, the minimal layout was added and now sections should look better in there

Arc_2024-07-09 13-16-29@2x
Arc_2024-07-09 13-16-48@2x
Arc_2024-07-09 13-17-13@2x
Arc_2024-07-09 13-17-51@2x

### CODE REVIEW

File: templates/index.json [hunk 1]
```diff
@@ -236,6 +254,110 @@
...
+      "block_order": [],
+      "settings": {
+        "background_image": "booqable://assets/image-about-us-background.png",
+        "button_label": "MORE ABOUT US",
+        "button_url": "booqable://root",
+        "color_scheme": "set-3",
+        "description": "Our passion for producing amazing wedding experiences has taken us on a fantastic path, where we have perfected the art of translating treasured fantasies into a reality of elegance. element symbolises the pinnacle of luxury, where love and beauty are harmoniously intertwined.",
+        "full_width": true,
+        "image": "booqable://assets/image-about-us.png",
+        "image_circle": true,
```
Comment (dabrorius) [inline templates/index.json:350]: This option does not seem to have any effect on the section.
Screenshot 2024-07-15 at 12 58 13

Comment (dabrorius) [reply templates/index.json:350]: Ah it seems the problem is that the default image is cut out in a circle so you can't see any difference.
However, it does seem to mess up aspect ratio of other images.
Screenshot 2024-07-15 at 12 59 35

Comment (srgzlb) [reply templates/index.json:350]: Yes, this option cannot be used with all images, but it also allows you to split the screen in a 40/60% proportion (as in variant designs) and make bigger room for text

Comment (dabrorius) [reply templates/index.json:350]: Can we use [object-fit: cover](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) to make it usable with non-suqare images?

Comment (srgzlb) [reply templates/index.json:350]: Sure thing

**Review by dabrorius — APPROVED** [review]

**Pull request merged**